suite/auto-sync/src/autosync/cpptranslator/patches/AddCSDetail.py

```diff
@@ -70,14 +72,23 @@ def get_patch(self, captures: [(Node, str)], src: bytes, **kwargs) -> bytes:
 
         comp = captures[3][0]
         comp = get_text(src, comp.start_byte, comp.end_byte)
-        return b"void " + fcn_id + params + b"{ " + add_cs_detail + comp.strip(b"{")
+        return (
```

Comment:
Good idea as well for now. In the long run the LLVM code will be it's complete own compilation unit and own interface. So we do not have all these annoying linking issues.